tests/utils/getUrl.test.ts

```diff
@@ -10,3 +29,53 @@ describe('Retrieving urls', () => {
     expect(getUrl('foo')).toBe(undefined)
   })
 })
+
+// Test getUrl(): per environment url overrides
+// Note: This tests against real content
+const cases = [
+  ['URL_PREFIX_UIO_DESKTOP', 'urlPrefixUioDesktop', 'uio.edd.ca.gov/UIO', 'uio-desktop-home'],
+  ['URL_PREFIX_UIO_MOBILE', 'urlPrefixUioMobile', 'uiom.edd.ca.gov/UIOM', 'uio-mobile-home'],
+  ['URL_PREFIX_UIO_CLAIMSTATUS', 'urlPrefixUioClaimstatus', 'uio', 'uio-claimstatus'],
+  ['URL_PREFIX_BPO', 'urlPrefixBpo', 'portal.edd.ca.gov/WebApp', 'bpo-login'],
+]
+describe.each(cases)(
+  'Retrieving urls that differ between EDD environments',
+  (envVar: string, key: string, toReplace: string, linkKey: string) => {
+    const mockedPrefix = 'mock-prefix'
+    const argPrefix = 'arg-prefix'
+
+    it(`returns the url with env var prefix when the env var is provided for ${envVar}`, () => {
+      const restore = mockEnv({
+        [envVar]: mockedPrefix,
+      })
+
+      const url = getUrl(linkKey)
+      expect(url).toEqual(expect.stringMatching(mockedPrefix))
```

Comment:
Shouldn't matter in practice but unless we expect argPrefix or mockedPrefix or toReplace to be regular expressions, I think it makes sense to use stringContaining instead of stringMatching everywhere in this section? [docs](https://jest-bot.github.io/jest/docs/expect.html#expectstringcontainingstring)

Reply:
I don't think that's going to work because `stringContaining()` is for exact matches, whereas `stringMatching()` is for partial matches. From these [docs](https://jestjs.io/docs/expect#expectstringcontainingstring):

> expect.stringContaining(string) matches the received value if it is a string that contains the exact expected string.

> expect.stringMatching(string | regexp) matches the received value if it is a string that matches the expected string or regular expression.

Also, I'm a little wigged out there there is both documentation at https://jest-bot.github.io/jest/ and https://jestjs.io. I assume that jestjs.io is more up to date and official?

Reply:
By "a string that contains the exact expected string" it doesn't mean exact matches, it means partial matches. (I think the docs should drop the word "exact" because it only adds confusion/ambiguity)

I tested this by replacing:
```
      const url = getUrl(linkKey)
      expect(url).toEqual(expect.stringMatching(mockedPrefix))
```
with:
```
      const url = 'mock-prefix-blah'
      expect(url).toEqual(expect.stringContaining(mockedPrefix))
```
and the test still passes when running `yarn test tests/utils/getUrl.test.ts` which is what you'd expect if stringContaining is for partial matches

(Oh yeah, jestjs.io looks more official! I was top-result googling 😅)

Reply:
I am saddened by the language in the jest docs 😞

Reply:
Resolved in #568